### ISSUE TITLE
scss/function-no-unknown triggers with scoped functions of SCSS modules

### ISSUE THREAD

**Issue by masi** [stylelint-scss/stylelint-scss#760]
```
@use 'mymodule';

.foobar {
  property: mymodule.myfunction();
}
```

This code should not trigger the rule.

**Comment by kristerkari** [comment]
Sounds like a bug!

**Comment by SaroGFX** [comment]
Same issue

**Comment by masi** [comment]
Workaround:

`'scss/function-no-unknown': [ true, { ignoreFunctions: /\./ } ],`

**Comment by kristerkari** [comment]
I opened a Pull Request to add support for `@use` and namespaces. Please let me know if I missed something: https://github.com/stylelint-scss/stylelint-scss/pull/773
